sdk/core/azure-core/vcpkg/README.md

```diff
@@ -0,0 +1,4 @@
+# Vcpkg Publish Automation Assets
+
+These files are used by the engineering system to publish to vcpkg. They are not
+required to build the library.
```

Comment:
nit: keep these as a single line since it is one paragraph and let GitHub do any line wrapping it needs to.

```suggestion
These files are used by the engineering system to publish to vcpkg. They are not required to build the library.
```